### DISCUSSION TITLE
雷达图"起始角度偏移"，在 grid、circle 坐标下表现不一致

### DISCUSSION THREAD

**Discussion by ntscshen** [antvis/G2Plot#2297] [Q&A]
没有没有计划，在雷达图中新增"起始角度偏移"api，其他圆形图都存在类似的操作

**Comment by hustcc** [comment]
可以加，也建议这些相对简单一些的，你们也可以参入进来，熟悉底层代码，对你们后续使用，也有很大用户。

**Comment by visiky** [comment]
参考这个PR: https://github.com/antvis/G2Plot/pull/2164, 把极坐标系下的图表都帮忙加一下？(雷达图、玫瑰图等，一个 plot 一个 pr)

**checklist:**

- [x] `types`
- [x] `adaptor`
- [x] `__tests__/${plot}/coodinate-spec.ts`
- [x] API 文档: `docs/api/plots/${plot}.zh.md` & `docs/api/plots/${plot}.en.md`

**Comment by ntscshen** [comment]
@visiky 在添加完成后测出一个可能是个问题的操作，https://codepen.io/ntscshen/pen/bGBpYxV?editors=1010
